docs/how-to-work-on-guide-topics.md

```diff
@@ -216,78 +224,78 @@ If there are other Guide resources you think campers would benefit from, add the
 - [A New Resource](#link)
 ```
 
-## Formatting
+#### Formatting
 
 Use double quotes where applicable.
 
 Format language keywords as code - this is done with the backtick key (located to the left of the "1" key on a US keyboard) in GitHub-flavored markdown. For example, put back ticks around HTML tag names or CSS property names.
 
 Use the Oxford Comma when possible (it is a comma used after the penultimate item in a list of three or more items, before ‘and’ or ‘or’ e.g. an Italian painter, sculptor, and architect). It makes things easier, clearer, and prettier to read.
 
-## Technical Writing
+#### Technical Writing
 
 Technical writing, or the literature of science and technology, is hard.
 
 You'll need to take a technical (usually abstract) topic and explain it in a clear, accurate, and objective manner.
 
 You'll likely go through several rounds of proofreading and editing before you're happy with the result.
 
-## Outline
+##### Outline
 
 Before you begin writing, create an outline of the topic and think about any coding examples you'll use (if applicable).
 
 This helps to organize your thoughts and make the writing process easier.
 
-## Intro
+##### Intro
 
-The introduction paragraph should only be 1-2 sentences long and be a simple explanation of the main topic. It should limit the use of any links to other Guide articles, as they can be distracting.
+The introduction paragraph should only be 1-2 sentences long and be a simple explanation of the main topic. It should limit the use of any links to other Guide topics, as they can be distracting.
 
-## Content
+##### Content
 
 Keep paragraphs short (around 1-4 sentences). People are more likely to read several short paragraphs over a wall of text.
 
-## Clarity
+##### Clarity
 
-Articles should be written with short, clear sentences, and use as little jargon as necessary.
+topics should be written with short, clear sentences, and use as little jargon as necessary.
 
 All jargon should be defined immediately in plain English.
 
-## Voice
+##### Voice
 
 Use active voice instead of passive voice. Generally, it's a stronger and more straightforward way to communicate a subject. For example:
 
-### Passive
+##### Passive
 
 The `for` loop in JavaScript is used by programmers to...
 
-### Active
+##### Active
 
 Programmers use the `for` loop in JavaScript to...
 
-## Point of View
+##### Point of View
 
 Text should use the second person ("you") to help to give it a conversational tone.
```

Comment:
```suggestion
The text should use the second person ("you") to help to give it a conversational tone.
```

Reply:
I think the "The" can be left off.

Reply:
I suggest we add it.
What do you think @moT01 @ojeytonwilliams ?

Reply:
It's marginal, but I prefer simply "Text" as it's being used to refer to text generally rather than a specific instance of it.